pkg/ipam/ipam.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+// ExecAdd invoke the plugin to add container to network
```

Comment:
hm but maybe we should be talking about the current process instead of "container"?